eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,6 +203,30 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if runtime dependency is absent.
+     * @param deps Dependencies
+     *  fixme: We are using hardcoded version of RuntimeDependency.
```

Comment:
I've tried to create a puzzle, but it complains that I have to link to some issue. As far as I know, I have to put current issue [#35](https://github.com/objectionary/eoc/issues/35), but we are not in `eoc` repository. `eo` repository has it's own #35 issue.

Reply:
> I've tried to create a puzzle, but it complains that I have to link to some issue. As far as I know, I have to put current issue [#35](https://github.com/objectionary/eoc/issues/35), but we are not in `eoc` repository. `eo` repository has it's own #35 issue.

In this case we must create new issue within `eo` repository since it is a fix for EO.

Reply:
New issue #1361

Reply:
Fixed